docs/docs/refs-and-the-dom.md

```diff
@@ -73,7 +73,23 @@ class AutoFocusTextInput extends React.Component {
 }
 ```
 
-You may not use the `ref` attribute on functional components because they don't have instances. You can, however, use the `ref` attribute inside the functional component:
+**You may not use the `ref` attribute on functional components** because they don't have instances. In the below code the ref will always be called with `null`:
+
+```
+function StatelessComponent() {
+  return <div>hello</div>
+}
+
+class ParentComponent extends React.Component {
+  render() {
+    return <StatelessComponent ref={node => this.helloNode = node} />
+  }
+}
+```
+
+If you need to a ref to a component, it must be a class (additional discussion can be found in [this issue on GitHub](https://github.com/facebook/react/issues/4936)).
```

Comment:
I don't think it's necessary to link to GH here as if it was a bug. People who are interested will find it anyway (you did).